cached.php

```diff
@@ -22,7 +22,7 @@
 $abs = realpath($abs);
 
 if (strncmp($abs, $pwd, strlen($pwd)) != 0) {
-    header("Location: https://www.php.net/" . strtr($_GET["f"],array("\r"=>"","\n"=>"")));
+    header("Location: https://www.php.net/" . strtr($_GET["f"],array("\r"=>"", "\n"=>"")));
```

Comment:
Huh, shouldn't this be
```suggestion
    header("Location: https://www.php.net/" . strtr($_GET["f"], array("\r"=>"", "\n"=>"")));
```
If so, is there maybe a bug in the fixer?

Reply:
@cmb69 

See https://github.com/php/web-php/pull/640!